.changeset/real-plums-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-auth-backend': minor
```

Comment:
This'll be a `patch` change as it should be backwards compatible and we're on `0.x`.